docs/Plugin-sap.commerce.build.md

```diff
@@ -31,14 +39,33 @@ hybris {
    
    //Default task dependencies for all tasks generated by the hybris ant rule
    antTaskDependencies = []
+   
+   //Control the sparse platform bootstrap.
+   //  When enabled, the commerce extensions are extracted from the distribution zip on the need basis.
+   //  Only extensions that are actually used in the project (either directly mentioned in the localextensions.xml or used by other
```

Comment:
„Only extensions that are actually used in the project (either directly listed in the localextensions.xml or required by other extensions) are extracted.“

Reply:
Changed